### ISSUE TITLE
drop scikit-learn 0.17.x support

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#50]
I tried to add tests for scikit-learn 0.17, but it turns out compatibility shims in eli5.lime don't work - e.g. KFold has different API. What do you think about dropping scikit-learn 0.17 support, and supporting only 0.18.x? //cc @lopuhin

**Comment by lopuhin** [comment]
I think that's fine to drop it @kmike . I checked eli5.sklearn tests: they pass on 0.17.1, but I don't think supporting it is worth the complexity.
